assets/pages/subscriptions-landing/components/countrySwitcherHeaderContainer.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+
+// ----- Imports ----- //
+
+import { connect } from 'react-redux';
+
+import CountrySwitcherHeader from 'components/headers/countrySwitcherHeader/countrySwitcherHeader';
+
+import type { CountryGroupId } from 'helpers/internationalisation/countryGroup';
+import type { State } from '../subscriptionsLandingReducer';
+
+
+const availableCountriesGroups: CountryGroupId[] =
+  ['GBPCountries', 'UnitedStates', 'EURCountries', 'AUDCountries', 'NZDCountries', 'Canada', 'International'];
+
+// ----- Functions ----- //
+
+function handleCountryGroupChange(value: string): void {
+  switch (value) {
+    case 'UnitedStates':
+      window.location.pathname = '/us/contribute';
+      break;
+    case 'AUDCountries':
+      window.location.pathname = '/au';
+      break;
```

Comment:
Is the AU page still active now that the test is over?

Reply:
perhaps not - we'll remove in a future PR though

Reply:
Don't worry about this.